lib/PhasarLLVM/DataFlowSolver/IfdsIde/IFDSFieldSensTaintAnalysis/Utils/DataFlowUtils.cpp

```diff
@@ -24,8 +24,9 @@
 
 using namespace psr;
 
-static const llvm::Value *PoisonPill = reinterpret_cast<const llvm::Value *>(
-    "all i need is a unique llvm::Value ptr...");
+static const llvm::Value *PoisonPill = // NOLINT
+    reinterpret_cast<const llvm::Value *>(
+        "all i need is a unique llvm::Value ptr...");
```

Comment:
What the **** is this? Casting a string literal to an `llvm::Value *`?

![image](https://user-images.githubusercontent.com/3064168/147404344-90b6534e-c31e-4a12-9f4e-ea84f4675fad.png)

Reply:
Yeah, sorry. The analysis has been developed independently by another research group.